onadata/libs/pagination.py

```diff
@@ -57,3 +62,73 @@ def paginate_queryset(self, queryset, request, view, count=None):
 
         self.request = request
         return list(self.page)
+
+
+def generate_pagination_headers(
+    request: Request, no_of_records: int, current_page_size: Union[int, str],
+    current_page: Union[int, str]
+) -> dict:
```

Comment:
Could we maybe add docstings for this helper function?
I am also curious, with regards to the `current_page_size` and `current_page` parameters we pass to this function. Why/When would we expect this to get returned as a string? I see from the tests that they are passed in as ints, also in the same function on line 93 and 99, we specifically convert them to ints, so why the Union?

Reply:
In some cases(especially when the value is retrieved from `request.query_param` the value is a str; this is just a way to support that it's not really necessary to have though). I'll add a docstring to this